codebase/superdarn/src.bin/tk/tool/find_freqband.1.0/doc/find_freqband.doc.xml

```diff
@@ -0,0 +1,91 @@
+<syntax>find_freqband  -cfit [<ar>options</ar>] <ar>cfitnames...</ar></syntax>
+<syntax>find_freqband  -cfit -old [<ar>options</ar>] <ar>cfitnames...</ar></syntax>
+
+<option><on>--help</on><od>print the help message and exit.</od>
+</option>
+<option><on>--version</on><od>print the RST version number and exit.</od>
+</option>
+<option><on>-vb</on><od>verbose. Log information to the console.</od>
+</option>
+<option><on>-sd <ar>yyyymmdd</ar></on><od>start from the date <ar>yyyymmdd</ar>.</od>
+</option>
+<option><on>-st <ar>hr:mn</ar></on><od>start from the time <ar>hr:mn</ar>.</od>
+</option>
+<option><on>-ed <ar>yyyymmdd</ar></on><od>stop at the date <ar>yyyymmdd</ar>.</od>
+</option>
+<option><on>-et <ar>hr:mn</ar></on><od>stop at the time <ar>hr:mn</ar>.</od>
+</option>
+<option><on>-ex <ar>hr:mn</ar></on><od>use the interval whose extent is <ar>hr:mn</ar>.</od>
+</option>
+<option><on>-tl <ar>tlen</ar></on><od>ignore the scan flag and instead use a fixed length scan of <ar>tlen</ar> seconds.</od>
+</option>
+<option><on>-cn <ar>channel</ar></on><od>process data from channel <ar>channel</ar> for stereo mode data.</od>
+</option>
+<option><on>-cn_fix <ar>channel</ar></on><od>manually set <ar>channel</ar> information written to output file.</od>
+</option>
+<option><on>-ns</on><od>exclude data with scan flags less than zero.</od>
+</option>
+<option><on>-bandwidth</on><od>manually specify the desired bandwidth in kHz (default=300 kHz).</od>
+</option>
+<option><on><ar>fitacfname</ar></on><od>filename of the <code>fitacf</code> format file.</od>
+</option>
+<option><on><ar>inxname</ar></on><od>filename of the index file.</od>
+</option>
+<option><on>-old</on><od>the input file is in the <code>fit</code> format, and the output should be in <code>grd</code> format. (If the input format is in <code>cfit</code> format as specified by the -cfit option, then the output will be in <code>grd</code> format).</od>
+</option>
+<option><on><ar>fitname</ar></on><od>filename of the <code>fit</code> format file.</od>
+</option>
+<option><on>-cfit</on><od>the input file is in the <code>cfit</code> format.</od>
+</option>
+<option><on><ar>cfitfname</ar></on><od>filename of the <code>cfit</code> format file.</od>
+</option>
+<option><on>-c</on><od>concatenate multiple input files (from the same radar site). WARNING: This option will be deprecated in the next RST release. Multiple input files from one radar are now detected and concatenated automatically.</od>
+</option>
+<option><on><ar>fitnames</ar></on><od>filenames of the <code>fit</code> format files.</od>
+</option>
+<option><on><ar>fitacfnames</ar></on><od>filenames of the <code>fitacf</code> format files.</od>
+</option>
+<option><on><ar>cfitnames</ar></on><od>filenames of the <code>cfit</code> format files.</od>
+</option>
+<synopsis><p>Creates a <code>grid</code> format file for a single radar site from either a <code>fit</code>, <code>fitacf</code> or <code>cfit</code> format file.</p></synopsis>
```

Comment:
There's quite a few references to the `make_grid` routine and command line options in this file. Should these be removed?

Reply:
Arg... probably not!